lib/activeadmin_addons/addons/markdown_builder.rb

```diff
@@ -0,0 +1,38 @@
+require 'redcarpet'
+
+module ActiveAdminAddons
+  class MarkdownBuilder < CustomBuilder
+    def render
+      return if data.blank?
+
+      Redcarpet::Markdown.new(renderer, extensions).render(@data).html_safe
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Tagging a string as html safe may be a security risk.